Sources/Common/Service/UserAgentUtil.swift

```diff
@@ -23,7 +23,7 @@ public class UserAgentUtilImpl: UserAgentUtil {
      * `Customer.io iOS Client/1.0.0-alpha.16`
      */
     public func getUserAgentHeaderValue() -> String {
-        var userAgent = "Customer.io iOS Client/\(deviceInfo.sdkVersion)"
+        var userAgent = "Customer.io \(deviceInfo.osName ?? "iOS") Client/\(deviceInfo.sdkVersion)"
```

Comment:
I haven't checked the documentation yet, but we need to be certain the only possible values from `osName` are `iOS` and `visionOS` otherwise our metrics can be skewed.

Reply:
[Docs are pretty generic](https://developer.apple.com/documentation/uikit/uidevice/1620054-systemname) unfortunately. 

I think the only way we can verify `deviceInfo.osName` is with QA testing. Print `deviceInfo.osName` to the console in iOS sample app and then in visionOS app and see what value gets printed.

Reply:
We can also use some constraints that ensures the `.osName` always returns a valid value. Realistically speaking though, if we are restricting our packages to iOS and VisionOS only, I don't see scenario where a third value comes in.